src/components/Book.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { AiOutlineLoading3Quarters } from 'react-icons/ai';
+import { removeBookFetch } from '../redux/books/books';
+import classes from './Book.module.css';
+
+const Book = (props) => {
+  const { id, book } = props;
+  const { title, author, category } = book;
+  const dispatch = useDispatch();
+
+  const removeBook = () => {
+    dispatch(removeBookFetch(id));
+  };
+  const num1 = Math.floor(Math.random() * 100);
+  const num2 = Math.floor(Math.random() * 20);
+  function getText() {
+    const sentences = [
+      'introduction',
+      'A Lesson Learned',
+      'principles of redux',
+    ];
+    const random = sentences[Math.floor(Math.random() * sentences.length)];
+    return random;
+  }
+
+  return (
+    <main className={classes.books}>
+      <div>
+        <li>
+          <div className={classes.book__list}>
+            <h3 className={classes.book__categories}>{category}</h3>
+            <h2>{title}</h2>
+            <p className={classes.book__author}>{author}</p>
+          </div>
+          <button type="button" className={classes.comment__btns}>
+            Comments
+          </button>
+          <button
+            type="button"
+            onClick={removeBook}
+            className={classes.remove__btns}
+          >
+            Remove
+          </button>
+          <button type="button" className={classes.edit__btns}>
+            Edit
+          </button>
+        </li>
+      </div>
+      <div className={classes.middle__section}>
+        <div className={classes.react__icon}>
+          <AiOutlineLoading3Quarters />
+        </div>
+        <div className={classes.middle__section_left}>
+          <p>
+            {num1}
+            <span>&#37;</span>
+          </p>
+          <p className={classes.completed}>Completed</p>
+        </div>
+      </div>
+      <div className={classes.right__section}>
+        <h1 className={classes.current__chapter}>
+          current
+          <span>&nbsp;</span>
+          chapter
+        </h1>
+        <p>
+          Chapter
+          <span>&nbsp; &nbsp;</span>
+          {num2}
+          <span>&nbsp; &nbsp;</span>
+          {getText()}
+        </p>
+        <button type="button" className={classes.update__btn}>
+          update progress
+        </button>
```

Comment:
- [ ] Kindly note that this button should look like the button in the template 
Template button
![image](https://user-images.githubusercontent.com/65068771/225773481-4d88c336-81ff-443c-ac6d-3a46d01e455a.png)
Yout button
![image](https://user-images.githubusercontent.com/65068771/225773555-ceaafc47-6fa1-462d-a880-eaac4324a746.png)

Apply the same styling for you submit button
 
![image](https://user-images.githubusercontent.com/65068771/225773879-b637a82f-eff8-437f-a973-dc071b6318a5.png)